src/metabase/util/malli/schema.clj

```diff
@@ -21,7 +21,7 @@
       ...)"
   [model]
   (mc/schema
-    [:fn {:error/fn (constantly (deferred-tru "value must be an instance of {0 " (name model)))}
+    [:fn {:error/fn (constantly (deferred-tru "value must be an instance of {0}" (name model)))}
```

Comment:
you can't use `constantly` here since it evaluates its argument at compile time before we have a chance to have a locale. you need to use `(fn [] (deferred-tru ...))`

Reply:
that's overlooked. Thanks! I'll open a PR to do the same for all schema in this.